modules/clair-openshift-airgap-database-standalone.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// clair/master.adoc
+
+:_content-type: PROCEDURE
+[id="clair-openshift-airgap-database-standalone"]
+= Configuring access to the Clair database in the disconnected {ocp} cluster
+
+Use the following procedure to configure access to the Clair database in your disconnected {ocp} cluster.
+
+.Prerequisites
+
+* You have installed the `clairctl` command line utility tool.
+* Optional. If your Clair deployment is provisioned by {ocp}, you have retrieved and decoded the Clair configuration secret, and saved it to a Clair `config.yaml` file.
+* Optional. If you are configuring standalone Clair on a a disconnected {ocp} cluster, you have deployed Clair.
+*  The `disable_updaters` and `airgap` parameters are set to `true` in your Clair `config.yaml` file.
+* You have exported the updaters bundle from a Clair instance that has access to the internet.
```

Comment:
Can we link to instructions for the process wherever this is called out?

Reply:
Not until after publication